updatecli/generate-manifests.ps1

```diff
@@ -18,8 +21,16 @@ if ([System.IO.Directory]::Exists($manifestDirectory) -eq $false) {
   New-Item -Force -ItemType Directory -Path $manifestDirectory | Out-Null
 }
 
-# bills would be sorted by name due to our naming convention of "bom-x.y.z"
-$bills = Get-ChildItem "$root/bom-*" -Directory | Select-Object -ExpandProperty Name
+# Get current bom versions based on directory structure afterwards sort by version number ascending
+$bills = Get-ChildItem "$root/bom-*" -Directory `
+  | Select-Object -ExpandProperty Name `
+  | Where-Object { $_ -ne "bom-weekly" } `
+  | ForEach-Object { [System.Version]$($_ -replace '^bom-(\d+\.\d+?)\.x$','$1') } `
+  | Sort-Object `
+  | ForEach-Object { "bom-$($_.Major).$($_.Minor).x" }
```

Comment:
Sort-Object by default follows alphanumeric order and there for use System.Version to get version order.

```pwsh
$a = @('1.1','1.2','1.10')
$a | Sort-Object          
1.1
1.10
1.2
```

vs

```pwsh
$a | ForEach-Object {[System.Version]$_} | Sort-Object | ForEach-Object {"$($_.Major).$($_.Minor)"}
1.1
1.2
1.10
```